cli/ioctl/cmd/action/action.go

```diff
@@ -133,6 +133,21 @@ func execute(contract string, amount *big.Int, bytecode []byte) (err error) {
 		log.L().Error("error when invoke an execution", zap.Error(err))
 		return
 	}
+	accountMeta, err := account.GetAccountMeta(signer)
```

Comment:
Can you avoid dup code, and make this check as a method?

Reply:
sure